cmd/zrepl/zrepl.c

```diff
@@ -301,14 +339,13 @@ uzfs_zvol_worker(void *arg)
 		case ZVOL_OPCODE_READ:
 			rc = uzfs_read_data(zinfo->zv,
 			    (char *)zio_cmd->buf,
-			    hdr->offset, hdr->len, NULL, NULL);
+			    hdr->offset, hdr->len,
+			    &zio_cmd->metadata_desc);
```

Comment:
we need to read metadata only when we are in degraded mode

Reply:
good catch! I will fix that